pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+Thank you for working on PD! Please read PD's [CONTRIBUTING](https://github.com/pingcap/pd/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
```

Comment:
- It'd be better to put this file in `.github/pull_request_template.md`. 
- For this line, please put this in an HTML comment. 

Ref: 
- https://github.com/pingcap/tikv/pull/3154
- https://github.com/pingcap/tikv/blob/master/.github/pull_request_template.md